### PR TITLE
added cpack support

### PR DESCRIPTION
**Pull request by matt2005**
Added CPack support to enalbe building of deb packages

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -77,3 +82,37 @@ include_directories(${BTSERVICE_PROTO_INCLUDE_DIRS})
 add_subdirectory(openauto)
 add_subdirectory(autoapp)
 add_dependencies(autoapp btservice_proto)
+
+install(DIRECTORY lib DESTINATION lib COMPONENT libraries)
+install(DIRECTORY include DESTINATION include COMPONENT headers)
+install(DIRECTORY bin DESTINATION bin COMPONENT applications)
+
+SET(CPACK_GENERATOR "DEB")
+SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "OpenCarDev") #required
```
Comment (rsjudka) [inline CMakeLists.txt]: should this (and the line below) be OpenCarDev?

i know the crankshaft openauto is aloooooot different and dont want any confusion

Comment (matt2005) [reply CMakeLists.txt]: Fixed the typo

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+project(openauto 
+LANGUAGES CXX)
+
 find_program(CCACHE_PROGRAM ccache)
 if(CCACHE_PROGRAM)
     set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
 endif()
-
-project(openauto CXX)
+set (OPENAUTO_VERSION_MAJOR 2)
```
Comment (rsjudka) [inline CMakeLists.txt]: some sort of merge error? looks like its being set twice

File: include/openauto/Service/InputService.hpp [hunk 3]
```diff
@@ -38,7 +38,7 @@ class InputService:
 public:
     InputService(boost::asio::io_service& ioService, aasdk::messenger::IMessenger::Pointer messenger, projection::IInputDevice::Pointer inputDevice);
 
-    void sendButtonPress(aasdk::proto::enums::ButtonCode::Enum buttonCode);
+    void sendButtonPress(aasdk::proto::enums::ButtonCode::Enum buttonCode, projection::WheelDirection wheelDirection = projection::WheelDirection::NONE);
```
Comment (rsjudka) [inline include/openauto/Service/InputService.hpp]: did you make these changes? i remember these being part of a different PR

Comment (matt2005) [reply include/openauto/Service/InputService.hpp]: Rebase messed up,  I'll have to rebase it again

**Pull request merged**